mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -234,6 +238,7 @@ CLASS ahl net/minecraft/entity/Entity
 	METHOD bN isImmuneToExplosion ()Z
 	METHOD bQ getPrimaryPassenger ()Lahl;
 	METHOD bR getPassengerList ()Ljava/util/List;
+	METHOD bS recursePassengers ()Ljava/util/Collection;
```

Comment:
getPassengers/getPassengersRecursively

Reply:
getPassengersDeep could also work (deep/shallow)

Reply:
I recall `getPassengersRecursively` is a voldemap name. Or `getAllPassengers`?